ref_ts/receivers.py

```diff
@@ -43,14 +53,13 @@ def ref_time_series_describe_resource_trigger(sender, **kwargs):
                                                                 soap_or_rest=ref_type,
                                                                 site_code=site_code,
                                                                 variable_code=variable_code)
-                del metadata[:]
                 ts_utils.prepare_metadata_list(metadata=metadata,
                                                ts_dict=ts_dict,
                                                url=ref_url,
                                                reference_type=ref_type)
             else:
                 raise Exception("Missing parameter 'referenceurl' in request")
 
-        except Exception as ex:
+        except BaseException as ex:
```

Comment:
Yes, _please_ don't go up to `BaseException`.  That will catch things such as keyboard interrupts, which is highly undesirable.